Makefile

```diff
@@ -30,7 +30,7 @@ lint:
 	@find . -name "*.py" -exec pylint -f colorized {} \;
 
 test:
-	@$(PYTEST) --cov=./xoinvader --cov-report=html --strict -v
+	@$(PYTEST) --cov=./xoinvader --cov-report=html --strict -v PYTEST_ARGS
```

Comment:
Should not it be `$(PYTEST_ARGS)`?
Also I don't quite know how `py.test` works, but from reading help page I suggest that `PYTEST_ARGS` will be interpreted as filename, which is not what we want.